app/javascript/dashboard/components/ChatList.vue

```diff
@@ -794,16 +794,16 @@ export default {
 
 .conversations-list-wrap {
   flex-shrink: 0;
-  width: 34rem;
+  flex-basis: 34rem;
   overflow: hidden;
   @include breakpoint(large up) {
-    width: 36rem;
+    flex-basis: 36rem;
   }
   @include breakpoint(xlarge up) {
-    width: 35rem;
+    flex-basis: 38rem;
   }
   @include breakpoint(xxlarge up) {
-    width: 38rem;
+    flex-basis: 42rem;
   }
   @include breakpoint(xxxlarge up) {
```

Comment:
Curious question? Are these values standardized or documented somewhere, if not we can use a formula-based width for this using a combination of calc and clamp.

```css
.conversations-list-wrap {
    flex-shrink: 0;
    /* Where 1vw is 1% of the viewport */
    flex-basis: clamp(34rem, 1vw + 34rem, 42rem);
}
``` 

Take this suggestion with a pinch of salt, **this would make the width of the list fluid**, and it can break in other places where we depend on the predictable and fixed widths of the list. The main benefit this adds is reducing the code footprint of the CSS we write

P.S. CSS `clamp` is widely supported, [ref](https://caniuse.com/css-math-functions)

Reply:
Done https://github.com/chatwoot/chatwoot/pull/6134/commits/f84b05b16cdc66f1241555c71f00c4f555e9301e